package.json

```diff
@@ -4,6 +4,7 @@
   "main": "expo-router/entry",
   "scripts": {
     "start": "expo start",
+    "both": "expo start --android --ios",
```

Comment:
is this just a script you use when you are testing on both platforms?

Reply:
Yes. I usually have both simulators running so I want to start them at the same time